lib/api/protocol/_base-action.js

```diff
@@ -172,6 +172,10 @@ module.exports = class ProtocolAction {
   executeScriptHandler(method, script, args, callback) {
     let fn;
 
+    if (script.originalTarget) {
+      script = script.originalTarget;
+    }
+
```

Comment:
Why is this needed?

Reply:
This is needed for executeScript to function. Proxy callbacks can't be converted to string.